diesel_infer_schema/src/information_schema.rs

```diff
@@ -29,6 +32,10 @@ impl UsesInformationSchema for Pg {
     fn type_column() -> Self::TypeColumn {
         self::information_schema::columns::udt_name
     }
+
+    fn default_schema<C>(_conn: &C) -> QueryResult<String> {
+        Ok("public".into())
```

Comment:
I'd love to see a `Cow` here. Not really important for the overall picture though.